### DISCUSSION TITLE
♥妮可不在家，猫猫偷偷看♥

### DISCUSSION THREAD

**Discussion by hao123nevergiveup** [share121/inter-knot#962] [其他]
猫猫要偷偷在被窝里看哦【[点击查看](https://www.marxists.org/chinese/index.html)】
![0c7a747249c562b9f812e1f18baac0781939354835](https://github.com/user-attachments/assets/f62c65b0-5c77-46ad-aef6-a3a9dfe7aff0)

**Comment by WLMRMZ** [comment]
同志！

**Comment by TrafficLight6** [comment]
省流：马克思文库

**Reply by hao123nevergiveup** [reply]
> 省流：马克思文库

[[我都挂梯子了，还省什么流](https://www.marxists.org/chinese/index.html)]

**Reply by uyfh352** [reply]
![螢幕擷取畫面 2024-08-10 200728](https://github.com/user-attachments/assets/d87cd15a-8746-4f0e-9de8-fe36cc807910)

**Comment by uyfh352** [comment]
![螢幕擷取畫面 2024-08-10 200728](https://github.com/user-attachments/assets/d9f2f9d8-91fd-457f-9030-db3fa6b70fd0)

**Comment by Yizakl** [comment]
👍

**Comment by Drink-H2O** [comment]
英特纳雄耐尔一定会实现

**Comment by Qiling-XwX** [comment]
![67B9034B2499AD7437F45A70803AEE8A](https://github.com/user-attachments/assets/59e10a79-2054-42e4-b803-35b4eb37956b)

**Comment by lllliufeng** [comment]
感谢楼主分享👍👍👍

**Comment by BITE-7274** [comment]
jm搜不到啊QwQ

**Comment by xingruxuewji** [comment]
> jm搜不到啊QwQ

@BITE-7274 

你是真饿了

**Comment by BuQiXiaokao** [comment]
不愧是你！同志
